tests/framework/OutStreams/sample_plot.xml

```diff
@@ -0,0 +1,78 @@
+<Simulation verbosity = 'debug'>
+
+  <TestInfo>
+    <name>framework/OutStreams.SamplePlot</name>
+    <author>talbpaul</author>
+    <created>2021-04-01</created>
+    <classesTested>OutStreams.SamplePlot</classesTested>
+    <description>
+      Tests use of the SamplePlot plotter.
```

Comment:
Could you expand the description here?

Reply:
It really doesn't do anything more than this; what would you like to see added?

Reply:
in the hopes it is sufficient to allay concerns, I added another sentence saying the same thing in different words to increase clarity.